source/CommonJobs/CommonJobs.Application.EvalForm/DTOs/CalificationsEvaluationDto.cs

```diff
@@ -35,7 +35,9 @@ public class CalificationsEvaluationDto
 
         public List<string> Evaluators { get; set; }
 
-        public static CalificationsEvaluationDto Create(EmployeeEvaluation evaluation, List<string> evaluators, string currentPosition, string seniority)
+        public bool IsCompanyEvaluationDone { get; set; }
```

Comment:
As disused in the call, I'll change the name of _ReadyToDevolution_ state if it's an easy change. A deeper analyzes may be required late to check states functions.